modules/itemManager.ts

```diff
@@ -102,8 +106,59 @@ class ItemManager {
   };
 
   /** Populate the list items in the main view, and the Section bubbles footer */
+  // async populateItems() {
```

Comment:
Get rid of this old code, and the logs we have in the ListManager or elsewhere before merge